x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/aggregate/PercentileAggregate.java

```diff
@@ -92,18 +93,18 @@ protected TypeResolution resolveType() {
             return resolution;
         }
 
-        resolution = isFoldable(parameter, sourceText(), ParamOrdinal.SECOND);
+        resolution = isFoldable(parameter, sourceText(), SECOND);
         if (resolution.unresolved()) {
             return resolution;
         }
 
-        resolution = isNumeric(parameter, sourceText(), ParamOrdinal.SECOND);
+        resolution = isNumeric(parameter, sourceText(), SECOND);
         if (resolution.unresolved()) {
             return resolution;
         }
 
-        ParamOrdinal methodOrdinal = ParamOrdinal.fromIndex(parameters().size() + 1);
-        ParamOrdinal methodParameterOrdinal = ParamOrdinal.fromIndex(parameters().size() + 2);
+        ParamOrdinal methodOrdinal = TypeResolutions.ParamOrdinal.fromIndex(parameters().size() + 1);
```

Comment:
same here, `import org.elasticsearch.xpack.ql.expression.TypeResolutions.ParamOrdinal` is already there.